### DISCUSSION TITLE
TuringCourses/major/cyberspace_security/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#23] [General]
# TuringCourses/major/cyberspace_security/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major/cyberspace_security/

**Comment by MYJOKERML** [comment]
想拿高分的话作业一定要认真对待，按照作业要求认真落实每一项任务，尤其是最后的结果一定要清清楚楚地呈现出来，不是自己的代码能跑出结果来就行了，要让老师能直接了当地看见你的结果，整理文档时条理要清晰。张秉晟的作业就分4个任务给的分，一旦漏了某个任务（结果呈现不清楚也可能被判漏）就会严重十分，又不考试的东西，时间也长，任务也不是很难，建议用心对待作业。

**Comment by Fairicle** [comment]
22 级的给分依然为 20% 平时成绩 + 80% 课程实践报告成绩，但是平时上课并没有进行点名，最后这 20% 的平时成绩是所有人都给满了的。

关于课程报告，22 级的选题如下，在三个方向中选择一个方向中的其中一个任务完成：
- 网络安全（以下两个二选一）
  - 完成一个 2000 字以上的领域（网络安全、物联网安全、多媒体安全）调研综述
  - 按照课程大作业形式提交一个自己主导的科研项目的报告，1000 字以上
- 系统安全（以下两个二选一）
  - 通过广泛查阅资料等各种方式，调研 Intel 或 ARM 处理器架构的安全机制，阐述其威胁模型、应用场景和基本工作原理，完成一个 2000 字左右的课程报告
  - 调研某一个有影响力的漏洞（有具体的漏洞编号），完成漏洞复现或分析工作，并阐述其技术原理，完成 1000 字的技术文档
- 数据安全（只有一个题目，并且有给定具体要求）
  - 差分隐私相关研究
